app/browser/menu.js

```diff
@@ -602,7 +606,10 @@ const createMenu = (state) => {
     })
   }
 
-  appActions.setMenubarTemplate(Immutable.fromJS(template))
+  if (isWindows) {
+    const menuTemplate = JSON.parse(JSON.stringify(template))
```

Comment:
I talked to Clifton, the handlers aren't actually used in app state and this is only called once.